### DISCUSSION TITLE
There is no option to select modded turrets in the perk menu

### DISCUSSION THREAD

**Discussion by AntiBlastYT** [FakeFishGames/Barotrauma#14772] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

While using the "THG Armory - 'Vanilla' Turret pack" i tried choosing one of the mods turrets in perk menu and there is no option to do so

### Reproduction steps

1. Start a multiplayer lobby
2. Select mission
3. Go to perk menu
4. No option for modded turrets11!!11

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Turrets don't show automatically in the perk menu: there needs to be a perk configured for buying it. I.e. the creator of the mod needs to add a perk that unlocks the turret.
